evdi_modeset.c:375:2: error: implicit declaration of function ‘drm_mode_create_dirty_info_property’

Getting this error when installing fedora-25-displaylink-1.2.64-1.x86_64.rpm
 on fedora 25 - 4.9.3-200.fc25.x86_64

        Manufacturer: Dell Inc.
        Product Name: XPS 13 9343
        00:02.0 VGA compatible controller: Intel Corporation HD Graphics 5500 (rev 09)

```
/var/lib/dkms/evdi/1.2.64/build/evdi_modeset.c: In function ‘evdi_modeset_init’:
/var/lib/dkms/evdi/1.2.64/build/evdi_modeset.c:375:2: error: implicit declaration of function ‘drm_mode_create_dirty_info_property’ [-Werror=implicit-function-declaration]
  drm_mode_create_dirty_info_property(dev);
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/var/lib/dkms/evdi/1.2.64/build/evdi_connector.c: In function ‘evdi_connector_init’:
/var/lib/dkms/evdi/1.2.64/build/evdi_connector.c:148:24: error: ‘struct drm_mode_config’ has no member named ‘dirty_info_property’; did you mean ‘tile_property’?
        dev->mode_config.dirty_info_property, 1);
                        ^
scripts/Makefile.build:293: recipe for target '/var/lib/dkms/evdi/1.2.64/build/evdi_connector.o' failed
make[2]: *** [/var/lib/dkms/evdi/1.2.64/build/evdi_connector.o] Error 1
make[2]: *** Waiting for unfinished jobs....
  if [ "-pg" = "-pg" ]; then if [ /var/lib/dkms/evdi/1.2.64/build/evdi_encoder.o != "scripts/mod/empty.o" ]; then ./scripts/recordmcount  "/var/lib/dkms/evdi/1.2.64/build/evdi_encoder.o"; fi; fi;
cc1: some warnings being treated as errors
scripts/Makefile.build:293: recipe for target '/var/lib/dkms/evdi/1.2.64/build/evdi_modeset.o' failed
make[2]: *** [/var/lib/dkms/evdi/1.2.64/build/evdi_modeset.o] Error 1
  if [ "-pg" = "-pg" ]; then if [ /var/lib/dkms/evdi/1.2.64/build/evdi_drv.o != "scripts/mod/empty.o" ]; then ./scripts/recordmcount  "/var/lib/dkms/evdi/1.2.64/build/evdi_drv.o"; fi; fi;
Makefile:1494: recipe for target '_module_/var/lib/dkms/evdi/1.2.64/build' failed
make[1]: *** [_module_/var/lib/dkms/evdi/1.2.64/build] Error 2
make[1]: Leaving directory '/usr/src/kernels/4.9.3-200.fc25.x86_64'
Makefile:21: recipe for target 'all' failed
make: *** [all] Error 2
```

possibly related to this:
https://github.com/DisplayLink/evdi/commit/b7606bccc0cb9484c2845a3e556faf60ed3c0bff

May need to go to this version of EVDI:
https://github.com/DisplayLink/evdi/releases/tag/v1.3.43

--- a/.travis.yml
+++ b/.travis.yml
@@ -2,8 +2,8 @@ language: generic
 sudo: required
 env:
   global:
-    - VERSION=1.2.64
-    - DAEMON_VERSION=1.2.1
+    - VERSION=1.3.43
+    - DAEMON_VERSION=1.2.65
     - RELEASE=1
   matrix:
   - OS_TYPE=fedora OS_VERSION=25

--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,5 @@
-VERSION=1.2.64
-DAEMON_VERSION=1.2.1
+VERSION=1.3.43
+DAEMON_VERSION=1.2.65
 DOWNLOAD_ID=708 # This id number comes off the link on the displaylink website
 RELEASE=1
 

--- a/README.md
+++ b/README.md
@@ -54,9 +54,6 @@ These variables need to be changed in the following places:
 - displaylink.spec
   - `%{daemon_version}` is the DisplayLinkManager version
   - `Version:` is currently the evdi driver version
-  - `Source4:` has to point to a valid zip name: change the advertised
-    version there
-
 - Makefile
   - `DAEMON_VERSION` is the advertised package version
   - `VERSION` is the same version as the specfile `Version:`

--- a/displaylink.spec
+++ b/displaylink.spec
@@ -2,7 +2,7 @@
 %define daemon_version 1.2.65
 
 Name:		displaylink
-Version:	1.2.64
+Version:	1.3.43
 Release:	1
 Summary:	DisplayLink VGA/HDMI driver for DL-5xxx, DL-41xx and DL-3xxx adapters
 
@@ -13,7 +13,7 @@ Source1:	displaylink.service
 Source2:	99-displaylink.rules
 Source3:        displaylink-sleep-extractor.sh
 # From http://www.displaylink.com/downloads/ubuntu.php
-Source4:	DisplayLink USB Graphics Software for Ubuntu 1.2.1.zip
+Source4:	DisplayLink USB Graphics Software for Ubuntu %{daemon_version}.zip
 ExclusiveArch:	i386 x86_64
 
 BuildRequires:	libdrm-devel

